bin/pull-latest-ccd-files.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-branchName=$1
+branchName=CIV-5174
```

Comment:
Should this not be changed back to $1?

Reply:
Thanks @m-meulendijks-v1 
Have updated it now